nursery/access-camera-in-dotnet-on-android.yml

```diff
@@ -0,0 +1,10 @@
+rule:
+  meta:
+    name: access camera in .NET on Android
+    namespace: host-interaction/hardware/camera
+    authors:
+      - michael.hunhoff@mandiant.com
+    scope: function
+  features:
+    - or:
+      - api: Android.Hardware.Camera::Open
```

Comment:
restrict to `os: android`? or maybe the name and API namespace are sufficient?

we'll have to come up with a consistent style on things like this. i don't know today.

Reply:
os and/or format restrictions make sense to add

Reply:
Discussed offline w/ @mr-tz . We'll leave this for now and continue discussion in the issue that I just created to track this.